src/text.rs

```diff
@@ -11,14 +13,34 @@ pub fn get_random_string() -> String {
     lipsum(random_number)
 }
 
-pub fn create_progress_bar(text: impl Into<Cow<'static, str>>, size: u64) -> ProgressBar {
+pub fn create_simulation_bar(total_ticks: usize) -> ProgressBar {
+    let progress_bar = ProgressBar::new(total_ticks.try_into().unwrap());
+    let style = ProgressStyle::default_bar()
+        .template(
+            "{prefix:>12.cyan.bold}: {spinner:.green} [{elapsed_precise}] [{wide_bar:.cyan/blue}] {pos:>7}/{len:7} ({eta})",
+        )
+        .progress_chars("=>-");
+    progress_bar.set_style(style);
+    progress_bar.set_prefix("Simulation");
+    let is_ci = env::var("CI").is_ok();
```

Comment:
I'm wondering if we could move this a level higher, as some kind of null/naive implementation instead of having these checks in the middle of the logic.